pg-node.sh

```
@@ -1,5 +1,28 @@
+    SHARED_LIB_DIR="/usr/local/lib/pasarguard-scripts/lib"
+fi
+
+for shared_lib in common.sh system.sh docker.sh github.sh; do
+    if [ ! -f "$SHARED_LIB_DIR/$shared_lib" ]; then
+        printf 'Missing shared library: %s\n' "$SHARED_LIB_DIR/$shared_lib" >&2
+        exit 1
+    fi
+done
+
+# shellcheck source=lib/common.sh
+source "$SHARED_LIB_DIR/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: lib/common.sh was not specified as input (see shellcheck -x).

(SC1091)